tests/cases/binit.py

```diff
@@ -2,10 +2,10 @@
 
 # http://rosettacode.org/wiki/Bin_given_limits
 
-from typing import List
```

Comment:
For the case where only one thing is imported, the old code was more concise.
Why not add a new test and leave these cases alone?

Reply:
It isnt more consise
```
from typing import List;List
import typing;typing.List
```
Not sure why you dont want to use the existing test cases.  A new cases just for this import is hardly worth it, and ive run out of time I can allocate to this project this weekend.